index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/15d46053c4.js" crossorigin="anonymous"></script>
+  <title>My Portfolio | Welcome</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="hero">
+    <nav>
+      <img src="images/My Logo.png" class="logo">
```

Comment:
- Let's wrap the image which is your logo in the anchor tag to help navigation from one part of your nav items to the other part of your portfolio.

Reply:
Fixed.